static/css/business.less

```diff
@@ -22,3 +22,17 @@
     });
 }
 
+.nav__small-business {
+    .expandable_header {
+        position: relative;
+    }
+    
+    .expandable_header-right {
+        .org-chart .node_expander();
```

Comment:
You can call a nested class as a mixin? Never knew that. That said, might want to break this out since it's no longer limited to `.org-chart`